### DISCUSSION TITLE
Saving Minecraft Splash Mod Problem

### DISCUSSION THREAD

**Discussion by PiziderKidGamer** [community/community#113222] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi! I can't edit the Minecraft Splashes on Geometry Dash, when I edit something and then hit save, it says it applies the changes but when I hit the X to leave the settings tab, it says: "**_Unsaved Changes_**" and I don't know what to do, can anyone help me?

**Comment by deyanoris** [comment]
It sounds like you're having trouble editing the Minecraft Splashes in Geometry Dash.

**Comment by deyanoris** [comment]
I've had a similar issue before. Have you tried refreshing the page or logging out and back in again? Sometimes that helps reset things. If that doesn't work, you might want to reach out to the Geometry Dash support team for assistance.

**Reply by PiziderKidGamer** [reply]
It’s ok, the problem was fixed a few weeks ago
